crates/rolldown_error/src/build_error/error_constructors.rs

```diff
@@ -176,6 +177,16 @@ impl BuildDiagnostic {
     Self::new_inner(ForbidConstAssign { filename, source, name, reference_span, re_assign_span })
   }
 
+  pub fn invalid_option(situation: &str, option: String) -> Self {
+    Self::new_inner(InvalidOption {
+      invalid_option_types: match situation {
+        "output.format" => InvalidOptionTypes::UnsupportedCodeSplittingFormat,
```

Comment:
Here is better using an enum to flag `invalidOptionWithUnsupportedCodeSplittingFormat`.

Reply:
Rollup also handles errors in a similar way, as seen [here](https://github.com/rollup/rollup/blob/186706b7229943c97e0f2742a2944c6a5d74b0ff/src/Bundle.ts#L237-L252). Given the similarity in logic, I've used the `InvalidOption` error with `UnsupportedCodeSplittingFormat`.

We can adopt a similar approach to Rollup: when supporting `output.file` and `output.sourcemapFile`, we can use the same error handling mechanism. This aligns with the multi-chunk output configuration checker.

Specifically, we can apply this error handling when dealing with `output.file` and `output.sourcemapFile`, ensuring this approach is consistent with the multi-chunk output configuration checker.